mods/ra/maps/exodus/exodus.lua

```diff
@@ -0,0 +1,394 @@
+Difficulty = Map.LobbyOption("difficulty")
+
+UnitsEvacuatedThreshold =
+{
+	hard = 200,
+	normal = 100,
+	easy = 50
+}
+
+AttackAtFrame =
+{
+	hard = 500,
+	normal = 500,
+	easy = 600
+}
+
+MinAttackAtFrame =
+{
+	hard = 200,
+	normal = 100,
+	easy = 50
```

Comment:
Copy paste mistake: https://github.com/OpenRA/OpenRA/blob/release-20131223/OpenRA.Mods.RA/Missions/Allies03Script.cs#L408 has the values I wanted.